src/features/cheatcodes/pages/AppComponents.tsx

```diff
@@ -370,6 +374,25 @@ export const AppComponents: FunctionComponent = () => {
           <OfferBackPlaceholder />
           <Text> - OfferBackPlaceholder </Text>
         </AlignedText>
+
+        <AlignedText>
+          <StepDots numberOfSteps={NUMBER_OF_STEPS} currentStep={currentStep} />
+          <Text> - Steps </Text>
+        </AlignedText>
+
+        <AlignedText>
+          <Button
+            title="Back"
+            onPress={() => setCurrentStep(currentStep === 1 ? currentStep : currentStep - 1)}
+          />
+          <Spacer.Column numberOfSpaces={2} />
+          <Button
+            title="Next"
+            onPress={() =>
+              setCurrentStep(currentStep === NUMBER_OF_STEPS ? currentStep : currentStep + 1)
```

Comment:
La condition sur `NUMBER_OF_STEPS` est toujours `true` :
```suggestion
              setCurrentStep(currentStep + 1)
```

Reply:
Pour serait-elle toujours true ? Si on je définis currentStep à 1 et NUMBER_OF_STEPS à 4 ce n'est pas le cas. J'ai mis en place cette condition pour qu'on ne puisse pas faire avancer la currentStep indéfiniment dans l'exemple mis en place dans AppComponents